main/secrets.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_secretsmanager_secret" "beekeeper-db" {
```

Comment:
Ah, OK, we use a secrets manager strategy to get the password out and into an env variable which Spring then looks for when connecting to the DB?

Reply:
Precisely